electricitylci/plant_water_use.py

```diff
@@ -201,6 +201,7 @@ def generate_plant_water_use(year):
         },
         inplace=True,
     )
+    final_water=final_water.drop(columns=["Electricity"])
```

Comment:
@m-jamieson Why is electricity being dropped?

Reply:
These were either NaN or the wrong value. Due to downstream processes these incorrect values or NaNs were the source of electricity data over more preferred sources, like EIA 923, and causing errors. 

One thing that just occurred to me is I'm not sure if there are implications for using this 2016-year data for other years...like if the total water use, derived from 2016, will get correctly scaled if paired with 2020-year electricity generation data.

Reply:
Okay. Just making sure this wan't a part of the NAICS filter. We'll have to look at the scaling of water use data in another issue.